src/cr/cube/measures/pairwise_pvalues.py

```diff
@@ -0,0 +1,74 @@
+# encoding: utf-8
+
+"""P-values of pairwise comparison or columns of a contingency table."""
+
+from __future__ import division
+
+import numpy as np
+
+from cr.cube.distributions.wishart import wishartCDF
+from cr.cube.util import lazyproperty
+
+try:
+    xrange
+except NameError:  # pragma: no cover
+    xrange = range
+
+
+class PairwisePvalues(object):
+    """Value object providing matrix of pairwise-comparison P-values"""
+
+    def __init__(self, slice_, axis=0):
+        self.slice = slice_
+        self.axis = axis
```

Comment:
ALL instance variables should be private, *especially* in a value object. If you leave them public, that encourages a caller/user to access them and also allows them to mutate them, which breaks the value-integrity of the object. I have not found a good reason to ever have a public instance attribute.

While you're at it, only *actually intended interface properties* should be public (lack a leading underscore). If any of the lazyproperties below are just for internal use, like serve interim calculation purposes, make them private too. Private until proven public is a good policy in my experience.